### ISSUE TITLE
Justicar upgraded Shaman Hero Power can become un-interactible

### ISSUE THREAD

**Issue by Patashu** [HearthSim/hs-bugs#113]
Player reports that he attacked with Maiden of the Lake, Dr. Boom, clicked his upgraded Shaman Hero Power and then ended his turn (presumably without selecting a totem). Afterwards, his Hero Power remained visually exhausted when his next turn began (but the opponent still saw it as usable). Because his prevents him from using it, this is also a gameplay bug.

https://www.reddit.com/r/hearthstone/comments/3idgqb/bug_justicar_bug_removes_your_hero_power/

**Comment by jleclanche** [comment]
Need to test what happens on reconnect.

**Comment by Patashu** [comment]
http://us.battle.net/hearthstone/en/forum/topic/20742164296

Maybe this is related?

"I just played a game, Priest vs. Priest, where my opponent disconnected (didn't play cards for 1 turn). Their d/c occurred in the middle of my turn. When they returned, my turn ended immediately, the card I was hovering (about to play) didn't play, and I was locked out of my hero power for the rest of the game (the hero power icon showed it as having been used and stayed locked like that). 

For the rest of the game, I couldn't use my hero power. Needless to say, I lost.

To summarize, my opponent d/c'd when they reconnected I couldn't use my hero power."

**Comment by Patashu** [comment]
https://www.youtube.com/watch?v=izur1VrPKic&t=25m40s

Video example. But StrifeCro can use the Hero Power, and the turn after that it visually flips over, so it's not a gameplay bug at least.

**Comment by culinko** [comment]
The StrifeCro's example is #162. I don't think it's the same as Totemic Slam, because it doesn't present you with any options to choose from.

**Comment by Patashu** [comment]
You're right, posting it over there.

**Comment by PattuX** [comment]
http://us.battle.net/hearthstone/en/forum/topic/20743154149

Related?

**Comment by culinko** [comment]
This didn't happen at the end of the turn, the Knife Juggler was still green and the end turn button was yellow. The sidebar didn't update and he hanged (unlike the shaman report).

**Comment by jleclanche** [comment]
Closing this, no confirmation in 2 major patches.
